### ISSUE TITLE
Unable to upgrade tools in anvil

### ISSUE THREAD

**Issue by kwestionmark** [SonarSonic/Calculator#40]
I have not been able to upgrade any tools in the anvil. Do not know if this a known issue or not, but have tested in both creative and survival worlds. but have not had any luck.

**Comment by SonarSonic** [comment]
Sorry, looks like I missed this issue. I'm not actually very familiar with Anvil mechanics what isn't working exactly?

**Comment by Tedyhere** [comment]
I'll take a peek Ollie and see what I can find on what exactly isn't working.  It might be with Modlife 3 and the Tool Utilities upgrades not working correctly, which they should since I assume you use the pickaxe/axe/shovel class for the tools

**Comment by SonarSonic** [comment]
Okay thanks :) Not even sure what this issue is, so quite difficult to fix

**Comment by hron84** [comment]
I checked vanilla mechanics (without Tool Utilities, not in Modlife 3) and I was able to upgrade a Redstone Pickaxe with an Efficiency I book. The pickaxe got the enchantment, and it works as expected except the glowing effect (but we're already discussing about it at #58). Also applied a Silk Touch enchantment with Enchantment Plus to this pickaxe and a lot of enchanments to my Redstone Sword (Sharpness V, Looting III, Unbreaking I) and these are working properly too. So I can double-confirm normal enchantments working as expected. 

Calculator 1.9.0
